client/parties/party-reducer.ts

```diff
@@ -25,9 +39,16 @@ export default keyedReducer(new PartyRecord(), {
 
     return new PartyRecord({
       id,
-      invites: Map(invites.map(i => [i.id, new PartyUserRecord(i)])),
-      members: Map(members.map(m => [m.id, new PartyUserRecord(m)])),
+      invites: OrderedMap(invites.map(i => [i.id, new PartyUserRecord(i)])),
+      members: OrderedMap(members.map(m => [m.id, new PartyUserRecord(m)])),
       leader: new PartyUserRecord(leader),
+      messages: List([
+        new SelfJoinPartyMessageRecord({
+          id: cuid(),
+          time: Date.now(),
```

Comment:
I know we decided to leave this in place for Chat because it was already like this, but now you're copy-pasting this to other stuff, please don't :(